packages/headless/src/features/facets/facet-search-set/category/category-facet-search-set-slice.ts

```diff
@@ -33,6 +36,12 @@ export const categoryFacetSearchSetReducer = createReducer(
       })
       .addCase(executeFacetSearch.fulfilled, (state, action) => {
         handleFacetSearchFulfilled(state, action.payload);
+      })
+      .addCase(clearFacetSearchResults, (state, {payload}) => {
```

Comment:
Let's add a unit test for this case and the specific facet search, checking that the `clearFacetSearchResults` action is being handled.